### PR TITLE
182089941 add logger testing

### PR DESCRIPTION
**Pull request by eireland**

### CODE REVIEW

File: v3/src/lib/logger.ts [hunk 1]
```diff
@@ -127,7 +125,6 @@ export class Logger {
     const event_value = JSON.stringify(args)
     const logMessage = this.createLogMessage(time, event, documentTitle, event_value, args)
     debugLog(DEBUG_LOGGER, "logMessage:", logMessage)
-    // sendToLoggingService(logMessage, this.stores.user)
     sendToLoggingService(logMessage)
     sendToAnalyticsService(event, category)
     // for (const listener of this.logListeners) {
```
Comment (tealefristoe) [inline v3/src/lib/logger.ts:130]: Remove these commented lines as well?

**Review by tealefristoe — CHANGES_REQUESTED** [review]
`logger.test.ts` is failing 😕

**Review by tealefristoe — APPROVED** [review]
Looks good 👍

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Pull request merged**